.github/workflows/test-create-venv.yml

```diff
@@ -0,0 +1,129 @@
+name: test-create-venv
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '**'
+  pull_request:
+    branches:
+    - '**'
+
+concurrency:
+  # SHA is added to the end if on `main` to let all main workflows run
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ github.event_name }}-${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release/') || startsWith(github.ref, 'refs/heads/long_lived/')) && github.sha || '' }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: ${{ matrix.os.name }} ${{ matrix.arch.name }} ${{ matrix.python.name }}
+    runs-on: ${{ matrix.os.runs-on[matrix.arch.matrix] }}
+    container: ${{ matrix.os.container[matrix.python.matrix] }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: macOS
+            matrix: macos
+            runs-on:
+              arm: [macOS, ARM64]
+              intel: [macos-latest]
+          - name: Ubuntu
+            matrix: ubuntu
+            runs-on:
+              arm: [Linux, ARM64]
+              intel: [ubuntu-latest]
+          - name: Docker Debian
+            matrix: docker-debian
+            runs-on:
+              arm: [Linux, ARM64]
+              intel: [ubuntu-latest]
+            container:
+              '3.7': docker://python:3.7-buster
+              '3.8': docker://python:3.8-buster
+              '3.9': docker://python:3.9-buster
+              '3.10': docker://python:3.10-buster
+              '3.11': docker://python:3.11-buster
+          - name: Windows
+            matrix: windows
+            runs-on:
+              intel: [windows-latest]
+        python:
+          - name: '3.7'
+            action: '3.7'
+            check: '3.7'
+            matrix: '3.7'
```

Comment:
https://github.com/Chia-Network/actions/actions/runs/3527762746/jobs/5917180165#step:4:76
```
------
.github/workflows/test-create-venv.yml:22:40: property access of object must be type of string but got "number" [expression]
   |
22 |     container: ${{ matrix.os.container[matrix.python.matrix] }}
   |                                        ^~~~~~~~~~~~~~~~~~~~~
.github/workflows/test-create-venv.yml:110:31: property access of object must be type of string but got "number" [expression]
    |
110 |       if: matrix.os.container[matrix.python.matrix] == ''
    |                               ^~~~~~~~~~~~~~~~~~~~~
------
```

@cmmarslender, do the single quotes not make for a string?

Reply:
hmm. I would expect that to be a string. I wonder if its just an issue with number strings.. Maybe we can call these like.. py3.7 or something if that doesn't break it somewhere else?

Reply:
https://github.com/Chia-Network/actions/pull/51/commits/5a9df1b1fbb0e473aeaf9a2feb4231548806627b  `*sigh*` yep.  So I guess the super linter is running that actionlint thing I mentioned (after William mentioned it to me) so I need to go report issues there and see if I'm able to help fix them.  Or...  it appears this is all your responsibility since they use go!  `:]`

https://github.com/rhysd/actionlint/issues/249
https://github.com/rhysd/actionlint/issues/250